.flake8

```
@@ -12,4 +12,7 @@
 # handled by pylint:
 # F841 = local variable is assigned to but never used
 ignore = E203,E301,E302,E305,E501,E701,E704,W503,F841
-select = E,F,W
\ No newline at end of file
+select = E,F,W
+
+exclude =
+  .venv
```

not 100% sure but i think these arent ignored by default either
```suggestion
  .venv
  .git,
  __pycache__
```

---

what an absolute joke